scipy/signal/fir_filter_design.py

```diff
@@ -708,22 +708,66 @@ def remez(numtaps, bands, desired, weight=None, Hz=None, type='bandpass',
 
     Examples
     --------
-    For a signal sampled at 100 Hz, we want to construct a filter with a
-    passband at 20-40 Hz, and stop bands at 0-10 Hz and 45-50 Hz. Note that
-    this means that the behavior in the frequency ranges between those bands
-    is unspecified and may overshoot.
+    In those examples the remez function gets used creating a bandpass-, bandstop-, low- and highpassfilter.
+    The used parameters are the filter order, an array with according frequency boundaries, the desired attenuation values and the sampling frequency.
+    Using signal.freqz the according frequency response gets calculated and plotted.
 
+    >>> import numpy as np
     >>> from scipy import signal
-    >>> fs = 100
-    >>> bpass = signal.remez(72, [0, 10, 20, 40, 45, 50], [0, 1, 0], fs=fs)
-    >>> freq, response = signal.freqz(bpass)
-
     >>> import matplotlib.pyplot as plt
-    >>> plt.semilogy(0.5*fs*freq/np.pi, np.abs(response), 'b-')
-    >>> plt.grid(alpha=0.25)
-    >>> plt.xlabel('Frequency (Hz)')
-    >>> plt.ylabel('Gain')
+
+    >>> def plot_response(fs, w, h, title):
+    >>>     plt.figure()
```

Comment:
I think these need to be `...` not `>>>`, see related error:

https://travis-ci.org/scipy/scipy/jobs/370506116#L1338